src/compiler/parser.ts

```diff
@@ -3078,6 +3087,7 @@ namespace Parser {
             case ParsingContext.VariableDeclarations:
             case ParsingContext.JSDocParameters:
             case ParsingContext.Parameters:
+            case ParsingContext.JSDocComment: // TODO(jakebailey): is it?

```

Comment:
Dangerous! This list probably is too conservative, but I definitely wouldn't risk it this late in the 5.0 release cycle.

Reply:
I'll drop it. I'm not sure either way. Not that I think reuse is actually working 100% anyway.